resources/assets/js/designer/components/modals/modal-create-report-add.vue

```diff
@@ -0,0 +1,330 @@
+<template>
+  <b-modal ref="modal" size="lg" @hidden="onHidden" centered title="Create Report Table">
+      <form>
+        <div class="form-group">
+          <label for="tableName" v-model="tableName">{{tableName}}</label>
+          <input type="text" class="form-control" id="tableName">
+        </div>
+
+        <div class="form-group">
+          <label for="description">{{description}}</label>
+          <textarea class="form-control" id="description" rows="3"></textarea>
+        </div>
+
+        <div class="d-flex justify-content-between">
+          <div class="form-group">
+            <label for="type" v-model="type">{{type}}</label>
```

Comment:
Does the label need to be reactive?

Reply:
thats just how I was asked to make the labels, can change if you want.